lib/seattleflu/id3c/cli/command/etl/redcap_det_swab_n_send.py

```diff
@@ -0,0 +1,575 @@
+"""
+Process REDCap DET documents into the relational warehouse.
+
+Contains some hard-coded logic for which project ID correlates to which project
+(e.g. 17421 is the PID for Shelters)
+"""
+import os
+import re
+import click
+import json
+import hashlib
+import requests
+from typing import Any, List, Dict, Optional, Match, Collection
+from datetime import datetime, timezone
+from id3c.db.session import DatabaseSession
+from id3c.db.datatypes import Json
+from id3c.cli.command.etl import etl, race
+
+from id3c.cli.command.etl.redcap_det import (
+    COMPLETE_AND_VERIFIED,
+    LOG,
+    HEADERS,
+    REVISION,
```

Comment:
Do we want to share the `REVISION` number for all of the projects?

Reply:
Same thoughts as above comment.

Reply:
Actually, I don't know the answer here. `mark_processed` requires `REVISION` so this is a trickier question.

Reply:
Same! An option would be to pass the `REVISION` number to `mark_processed`. Just not sure if we want to do that or just have the same number for all redcap-det etls

Reply:
I think each ETL should have it's own revision, else when only one of them changes, all of them will re-run on the entire DET log.

Reply:
Should we pass `REVISION` as an argument to `mark_processed` or create a new function within each ETL process?

Reply:
If we are using separate REVISION numbers then we also need to include ETL_NAME in the processing log

Reply:
> Should we pass `REVISION` as an argument to `mark_processed` or create a new function within each ETL process?

I don't think I have a strong preference.

> If we are using separate REVISION numbers then we also need to include ETL_NAME in the processing log

Yep!